.github/workflows/ci.yml

```diff
@@ -16,18 +16,8 @@ jobs:
 
       - run: yarn
       - run: yarn turbo run lint test
-      - run: npx vsce package
+      - run: yarn deps:build && yarn extension:compile && (cd apps/extension/client && npx vsce package --yarn)
```

Comment:
nit: splitting into separate runs would end up with a somewhat nicer log and more focused reports when some part of this fails